server/routes/getSavedSearches.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import {
+  IRouter,
+  IKibanaResponse,
+  ResponseError,
+} from '../../../../src/core/server';
+import { API_PREFIX } from '../../common';
+import { parseEsErrorResponse } from './utils/helpers';
+import { RequestParams } from '@elastic/elasticsearch';
+
+export default function (router: IRouter) {
+  router.get(
+    {
+      path: `${API_PREFIX}/getSavedSearch`,
+      validate: {},
+    },
+    async (
+      context,
+      request,
+      response
+    ): Promise<IKibanaResponse<any | ResponseError>> => {
+      const params: RequestParams.Search = {
+        index: '.kibana',
+        q: 'type:search',
+        size: 10000,
+      };
+      try {
+        const esResp = await context.core.elasticsearch.legacy.client.callAsInternalUser(
```

Comment:
it this required to be legacy.client?

Reply:
could you switch to adminClient, since we are now under 7.8

Reply:
Yes, forgot to merge in your PR- will do that